### PR TITLE
Beta Release Candidate 4 - including up to Development Release 4706

### PR DESCRIPTION
**Pull request by ZappoMan**
- Improvements to FPS, present rate, and simulation rate counters
- Save the set of running scripts when they change, as well as at shutdown
- Improve comfort of third-person HMD camera.
- Fix crash related to parented objects
- Rendering fixes for AMD graphics cards
- Rolled log files now included in crash reports
- Sandbox Agrents can now use atp urls for scripts
- Improved statistics for GPU usage
- Asset server cleans up unmapped files
- Fixes to saving changes to domain settings
- Golf club example
- No longer showing placeholder text in upload URL in edit tools
- Fixes to some crashes on shutdown
- Some render performance improvements
- New default LOD settings
- Fixes to context menus
- Support for displaying avatar licenses
- Smoother rendering for the Vive HMD
- New default home content
- Option to reset settings on startup

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 63468f8faa99bbec56e3c1e251870dfa2fd1091f available at: https://gist.github.com/9d9dacd3b7df6b140f24bbbd02ed8392<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7663.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7663.dmg

File: libraries/shared/src/Debug.h [hunk 1]
```diff
@@ -0,0 +1,249 @@
+//
```
Comment (jherico) [inline libraries/shared/src/Debug.h]: new files placed in the shared library should go in src/shared

**Comment by kevinmthomas-carpool** [comment]
## QA Test
**PR Build:** 7663
**Testing Architecture:** 64-bit PC Desktop
**Testing OS:** Windows 10
**Testing Locations:** localhost, playa, cellscience, forum
**Smoke Test Results:** Successful test.

                                  |localhost |playa    |cellscience |forum  
----------------------------------|----------|---------|------------|-----
Render Rate:                      |59.58     |58.94    |55.17       |54.26    
Present Rate:                     |60.04     |60.04    |60.04       |60.04   
Simrate:                          |59.00     |57.09    |56.10       |56.16    
Render Stats Num Buffers:         |900       |2787     |2640        |4646    
Render Stats gpu::Buffer Memory:  |21.4Mb    |35.8Mb   |23.7Mb      |53.1Mb        
Render Stats Num Textures:        |251       |520      |196         |776    
Render Stats gpu::Texture Memory: |652.2Mb   |1809.5Mb |399.3Mb     |1923.9Mb         
Render Stats Triangles:           |531k      |562k     |580k        |1388k    
Render Stats Drawcalls:           |170       |298      |204         |616   
Render Stats Items:               |242       |259      |807         |362
GPU-Z Memory Size                 |4096Mb    |4096Mb   |4096Mb      |4096Mb
GPU-Z Initial Memory		  |256Mb     |256Mb    |256Mb       |256Mb
GPU-Z Memory Used		  |1034Mb    |1973Mb   |867Mb       |1795Mb
GPU-Z Memory Load		  |8%        |11%      |7%          |10%

**localhost ‘createtestblocks.js’ Result:** Successful test.
**localhost ATP Upload (```kelectricguitar10.fbx```):** Successful test.
**localhost ‘harmonicOscillator.js’ Result:** Successful test.
**localhost Echo Server Audio Result:** Successful test.
**localhost ‘reverbTest.js’ Result:** Successful test.
**Additional Test Results Requested:** (none requested)
**Successful Logout/Crash:** Successful logout.

**Comment by OmegaHeron** [comment]
Built stack and Interface for Linux - tested deployed stack on Linux remote, all good.

**Comment by RyanDowne** [comment]
I looked at the content this morning and all is well

**Pull request merged**